### DISCUSSION TITLE
Assistance regarding GitHub Education Accreditation

### DISCUSSION THREAD

**Discussion by cosmog97** [community/community#139899] [GitHub Education]
### Hi everyone,

Hello!

I hope this message finds you well. I'm reaching out because I'm experiencing some issues with the accreditation process for GitHub Education, even though I believe I have met all the necessary requirements.

I have completed the application process, providing all the requested documentation and verifying my status as student. Despite this, my accreditation has been rejected, and I haven't received any specific feedback on what might be missing or incorrect.

Has anyone else encountered a similar issue? If so, I would greatly appreciate any advice or guidance on how to resolve this situation and successfully complete the accreditation process.

Thank you in advance for your help, and I look forward to any suggestions or recommendations you may have.

![PXL_20240926_205446815](https://github.com/user-attachments/assets/5e122624-3076-4cec-bb93-7e04f45ee55c)
![PXL_20240926_205434089](https://github.com/user-attachments/assets/a3901fa3-f643-4d8d-804b-28f4e5610ed1)
![99f3cbf0-fa67-4563-8335-ccfe8bb0a613-1](https://github.com/user-attachments/assets/4d67759c-07dc-482e-85aa-12641231617b)

**Comment by davevad93** [comment]
Hi @cosmog97 ,

I think the problem might be with the date on your certificate of enrollment which indicates the academic year `2023/2024` but the current academic year is `2024/2025`, it's true that it also indicates the date of issue, which is today `2024-09-26`, however one of the most common reasons of rejection is due to the dates.

The [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) state the following:

- ***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***

For example when I applied, I first submitted an enrollment certificate like you did which only showed the academic year and I was rejected, then, I submitted a screenshot of my last exam results which showed a current date in `DD/MM/YYYY` format and I was finally approved.

So try to do the same if possible or something similar, the most important thing is that the documentation you provide contains your full name, school's name and a current date in a `DD/MM/YYYY`, `MM/DD/YYYY` or `YYYY/MM/DD` format.
